docs_user/modules/proc_adopting-the-loadbalancer-service.adoc

```diff
@@ -5,39 +5,252 @@
 During the adoption process the {loadbalancer_first_ref} service
 must stay disabled in the new control plane.
 
-== Certificates
-Before running the script below the shell variables `CONTROLLER1_SSH` and
-`CONTROLLER1_SCP` must be set to contain the command to log into one of the
-controllers using `ssh` and `scp` respectively as root user as shown below.
+== Control plane adoption steps
 
+. Create an alias for the `openstack` command
++
 [source,bash]
 ----
-$ CONTROLLER1_SSH="ssh -i <path to the ssh key> root@192.168.122.100"
-$ CONTROLLER1_SCP="scp -i <path to the ssh key> root@192.168.122.100"
+$ alias openstack="oc exec -t openstackclient -- openstack"
```

Comment:
Would 'oc rsh openstackclient' be a better choice here?'

Reply:
Why? `oc rsh openstackclient` would create an interactive shell with that pod, so the openstack commands below would not work the same way. Also, lots of other services use the same alias, so using something else would be confusing.